ray-curator/ray_curator/models/aesthetics.py

```diff
@@ -118,3 +123,19 @@ def __call__(self, embeddings: torch.Tensor | npt.NDArray[np.float32]) -> torch.
         if isinstance(embeddings, np.ndarray):
             embeddings = torch.from_numpy(embeddings.copy())
         return self.mlp(embeddings.to(self.device)).squeeze(1)  # type: ignore[no-any-return]
+
+    @classmethod
+    def download_weights_on_node(cls, model_dir: str) -> None:
+        """Download the weights for the aesthetic scorer on the node."""
+        model_dir_path = Path(model_dir) / _AESTHETICS_MODEL_ID
+        model_dir_path.mkdir(parents=True, exist_ok=True)
+        model_file = model_dir_path / "model.safetensors"
+        if model_file.exists():
+            return
+        download_model_from_hf(
+            model_id=_AESTHETICS_MODEL_ID,
+            local_dir=model_dir_path,
+            filename="model.safetensors",
+            revision=_AESTHETICS_MODEL_REVISION,
```

Comment:
I see we use revision for CLIP, Aesthetic, Qwen but not for BERT, InternVideo and CosmosEmbed. Is there any reason for this? Can we make it consistent.